index.html

```diff
@@ -1691,7 +1691,72 @@ <h3>Add Proof</h3>
         </p>
 
       </section>
+      <section>
+        <h3>Add Proof Set</h3>
+        <p>
+          The following algorithm specifies how to add a proof set to a 
+          document. Required inputs are an
+          <a>unsecured data document</a> (<var>unsecuredDocument</var>) and a
+          (unordered) list of
+          <a>proof options</a> (<var>options_list</var>). Each <a>proof options</a>
+          (<var>options</var>) in the list must satisfy the criteria in 
```

Comment:
```suggestion
          <a>proof options</a> (<var>options_list</var>). Each <a>proof option</a>
          (<var>option</var>) in the list must satisfy the criteria in 
```

Reply:
The problem is that the "Add Proof" algorithm that this is using calls the input **proof options**, since there are multiple options to be dealt with per **proof**. Hence a bit of an ugly naming situation to deal with.

Reply:
I see no problem with the input being `proof options` and discussing each individual `proof option` which together comprise that plural input.